assets/less/base.less

```diff
@@ -37,9 +37,10 @@ body {
 .card-wrapper {
     padding: 5px;
 
-    @media (max-width: @screen-desktop) {
```

Comment:
Not sure the value this media query was providing; it was breaking for me in chrome and failing to float